var/spack/repos/builtin/packages/opengl/package.py

```diff
@@ -0,0 +1,56 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Opengl(Package):
```

Comment:
I also made this change at the request of @tgamblin, as I had originally named the package as you suggest.  I think @tgamblin preferred the shorter names for ease of typing?  I kind of preferred it this way, but @davydden pointed out there are no other cases where names are prepended with "system" like that.  I could easily go either way here.

Reply:
indeed, I suggest to name it `Opengl` as I don't think we have any other packages with `System` in their name:
```
$ spack list system
==> 1 packages.
lua-luafilesystem
```

I personally find it OK if `mesa` and friends provide `gl`, and reserve `opengl` for a system provided implementation of the interface.

Reply:
> we don't have any other packages with System in their name used in this sense:

That's true but AFAIK this is the first package that expects to be provided by the system, i.e. it raises an exception if you try to install it and it isn't provided as an external.

IMO it would be useful to identify such packages with a naming convention.

That being said I'd say hold off on this since I'm (and you're) the only one voting for `SystemOpengl` right now.